ratelimit/compat.py

```diff
@@ -0,0 +1,13 @@
+try:
```

Comment:
there's a reasonable argument that this whole file should just be `# flake8: ignore`d. that'll also fix the travis builds